packages/main/src/Select.ts

```diff
@@ -132,6 +132,22 @@ interface IOption extends UI5Element {
 		selectedOption: { type: HTMLElement },
 	},
 })
+/**
+ * Fired after the component's dropdown menu opens.
+ *
+ * @event sap.ui.webc.main.Select#open
+ * @public
+ * @native
+ */
+@event("open")
+/**
+ * Fired after the component's dropdown menu closes.
+ *
+ * @event sap.ui.webc.main.Select#close
+ * @public
+ * @native
```

Comment:
both events are not native. we are firing them with fireEvent which makes custom event